test/tstMinimumSpanningTreeGoldenTest.cpp

```diff
@@ -52,8 +52,8 @@ auto parsePointsFromCSVFile(std::string const &filename)
     Tokenizer tok(line);
     auto first = tok.begin();
     auto const last = tok.end();
-    points.emplace_back(ArborX::Point{std::stof(*first++), std::stof(*first++),
-                                      std::stof(*first++)});
+    points.emplace_back(std::stof(*first++), std::stof(*first++),
+                        std::stof(*first++));
```

Comment:
What happened her?

Reply:
`emplace_back` takes the arguments of the constructor. The changes here avoid calling the move constructor.